src/style.css

```diff
@@ -0,0 +1,138 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+  --main-color: #000;
+  --secondary-color: #777;
+  --moyo-color: #00acdc;
+}
+
+h2,
+h3,
+p {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
```

Comment:
Why you need flex in this case? Block elements by default located one belove the other.
Also don't repeat box-sizing: border-box, let's add this prop to all tag.
```
* {
  Box-sizing: Border-box
}
```

Reply:
Also, check the card's border. In code it's ok but in demo link no. Maybe you forgot deployed change.